Makefile

```diff
@@ -28,6 +28,10 @@ lint:
 lint-install:
 	./scripts/get-ktlint.sh
 
+# Generate public API binary
+generate-api:
+	./gradlew apiDump
```

Comment:
This generates the files, and the binary validator runs, and validates. So, previously we exposed `deviceToken` and now we changed it, it would have been a breaking change. So when I merged it, it gave me an error saying i am changing a publically exposed `API`. 

```
public fun getRegisteredDeviceToken ()Ljava/lang/String;
```

the change this command has created and is part of out `.api` file. Next time, someone changes it, it will match with this file and give an error.

Reply:
Oh, ok. 

You did good at explaining this. The names of the commands and the comments inside of the Makefile don't do as good of a job, IMO. 

A suggestion is changing the name of the commands in the Makefile:

```
generate-public-api: 
   ...

validate-public-api:
   ...
```

The way the file is now, I don't know what command to use, when. If commands explained the purpose, it could explain that you should run `generate-public-api` while developing a feature and run `validate-public-api` on CI to verify the stability of public-facing API.